molo/core/templatetags/core_tags.py

```diff
@@ -800,8 +800,7 @@ def social_media_article(context, page=None):
 @prometheus_query_count
 def get_next_article(context, article):
     locale_code = context.get('locale_code')
-    section = article.get_parent_section().get_main_language_page()
-    article = article.get_main_language_page()
```

Comment:
Was this line intentionally removed? I don't see any changes to this method that would require this?

Reply:
yes removed intentionally and working off of non main langauges

Reply:
whereas this was from main languages